addons/test_mail/tests/test_message_post.py

```diff
@@ -522,6 +522,22 @@ def test_message_log_with_view(self):
                 }
             )
 
+    def test_mail_message_notify_for_non_thread(self):
+        record = self.env['mail.template'].create({'name': 'Test1'})
+        message = self.env['mail.message'].create([{
+            'model': record._name,
+            'res_id': record.id,
+        }])
+        email_base_mail_values = self.env['mail.thread']._notify_by_email_get_base_mail_values(message)
+        self.assertMessageFields(
+            email_base_mail_values,
+            {
+                'mail_message_id': message.id,
+                'mail_server_id': False,
+                'references': message.message_id,
+                'subject': ''
+            }
+        )
```

Comment:
For future reference if you ever need to add tests in here again, the test itself here is fine but it should have been in TestMessageNotify, especially with the name you gave it. And we try to keep the methods in alphabetical order :)